pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm

```diff
@@ -106,14 +106,13 @@ sub generate_advice {
 
 sub _get_purchase_and_install_template {
     return << 'TEMPLATE';
-[%- locale.maketext('[asis,Imunify360] delivers sophisticated detection and display of security threats, powered by a self-learning firewall with herd immunity. It blocks attacks in real-time using a combination of technologies, including:') %]
+[%- locale.maketext('Use [asis,Imunify360] for a comprehensive suite of protection against attacks on your servers.') %]
```

Comment:
Ouch, we have to update strings in cP/WHM to make this work?  Makes no sense whatsoever this is in a separate repo.

Reply:
Do we have an automated way to pull in these string updates?

Reply:
Not to my knowledge.  cplint does not run lt checking on submodules.

Reply:
Thanks.  Tracking as CPANEL-31966